### ISSUE TITLE
Add Compyx

### ISSUE THREAD

**Issue by davelab6** [google/fonts#2959]
https://github.com/RoelN/Compyxz by @roeln

**Comment by enathu** [comment]
There is an extra character 'z' in the link -- https://github.com/RoelN/Compyx

:)

**Comment by enathu** [comment]
@davelab6 There is an extra character 'z' in the link -- https://github.com/RoelN/Compyx

:)

**Comment by RosaWagner** [comment]
Ok noted in my to do

**Comment by RosaWagner** [comment]
Ok, I my pipeline is a bit full for now, so Yanone takes over on this one

**Comment by RoelN** [comment]
I feel very honored, but this was kind of a joke towards @davelab6 when I ran into a 7-year old tweet: https://twitter.com/davelab6/status/487087874261536768

While the golden age of multicolor pixel fonts from the best selling legendary homecomputer in history _does_ make a great basis to show what color fonts can do, I would suggest a project that...

- Is based on an even more beautiful 8-bit demoscene typeface
- Is using COLR/CPAL
- Has a larger character set
- Has better source files and build process

No shortage of capable pixel artist, audiences that devour 80s cyber-aesthetic, and RoelNs who're foam-at-the-mouth enthusiastic about this, but as far as the original Compyx font is concerned — feel free to lower its priority in favor of better executed projects!

**Comment by davelab6** [comment]
@RoelN I am finally getting around to reviewing some fonts proposed under OFL long ago, and this is great - we've now launched color font support, so this can be onboarded :)

**Comment by RosaWagner** [comment]
@davelab6 there are 43 glyphs in this font :)

<img width="1407" alt="Capture d’écran 2023-06-01 à 11 17 19" src="https://github.com/google/fonts/assets/12222436/24052d00-419a-4148-b1f5-2308c6c8307e">

**Comment by RoelN** [comment]
Yes, the chararacter set is very limited. I would love to expand this so it has wider language support, and a better design! See the [older comment](https://github.com/google/fonts/issues/2959#issuecomment-765619279) :-)
